packages/server-boilerplate/src/microService/auth/middleware.ts

```diff
@@ -79,19 +74,21 @@ export const buildBasicBearerAuthMiddleware = (
       );
     }
 
-    let userId: string;
+    let accessPolicyObject: AccessPolicyObject;
     if (authHeader.startsWith('Bearer')) {
-      userId = await authenticateBearerAuthHeader(authHeader);
+      accessPolicyObject = await getBearerAccessPolicy(authenticator, authHeader);
     } else if (authHeader.startsWith('Basic')) {
-      userId = await authenticateBasicAuthHeader(req.models, authenticator, authHeader, apiName);
+      accessPolicyObject = await getBasicAccessPolicy(
+        req.models,
+        authenticator,
+        authHeader,
+        apiName,
+      );
     } else {
       throw new UnauthenticatedError('Could not authenticate with the provided access token');
     }
 
-    if (userId) {
```

Comment:
Seems to be a bit of a change from silently moving on to throwing an error here, which is probably a good thing, but just wanted to acknowledge it

Reply:
Yeah good point, though I do feel that we ought to be throwing an error here. A non-user request isn't valid right?

Reply:
Yeah I agree, I think it's a good change.